js/index.js

```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
+
+/***********
+INSTRUCTIONS:
+  - Select the project you would 
+    like to complete from the dropdown 
+    menu.
+  - Click the "RUN TESTS" button to
+    run the tests against the blank 
+    pen.
+  - Click the "TESTS" button to see 
+    the individual test cases. 
+    (should all be failing at first)
+  - Start coding! As you fulfill each
+    test case, you will see them go   
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
```

Comment:
### Your Review
![missing-why-2](https://user-images.githubusercontent.com/93039359/177151510-bcde5d53-c29e-4172-be4a-048b0152d4ca.png)

- [ ] You have suggested change here great. :+1: But you didn't tell the student why that student should apply this change. Please add a reason in the next review